src/a13-dates/test/index.test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-unused-vars */
+import assert from 'node:assert'
+import { getCurrentFormattedDate } from '../src/index.js'
+import { test, mock } from 'node:test'
+
+test('getCurrentFormattedDate returns the correct format', () => {
+  // Mock Date to a this timestamp: 2024-02-19T00:00:00Z
+  // Test the function with the mocked date
+  // assert.strictEqual(getCurrentFormattedDate(), '2024-02-19')
+  // Advance time to this specific date: 2025-12-25T00:00:00Z
+  // Test the function again with the new mocked date
+  // assert.strictEqual(getCurrentFormattedDate(), '2025-12-25')
```

Comment:
why these comments?

Reply:
These are used (also in other chapters) to better guide the user in the test case creation explaining what the user needs to do